experimenter/experimenter/experiments/api/v5/queries.py

```diff
@@ -32,5 +38,8 @@ def resolve_experiment_by_slug(self, info, slug):
         except NimbusExperiment.DoesNotExist:
             return None
 
+    def resolve_experiment_meta_by_application(self, info, application):
```

Comment:
Why `meta` in the name here?  I would expect this to be called something like `experiments_by_application`?